src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,17 @@
+<hibernate-configuration xmlns="http://www.hibernate.org/xsd/orm/cfg">
+    <session-factory>
+        <property name="dialect">org.hibernate.dialect.MySQLDialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/cinema_db</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">rootroot</property>
+        <property name="show_sql">true</property>
+        <property name="format_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.model.CinemaHall" />
+        <mapping class="mate.academy.model.Movie" />
+        <mapping class="mate.academy.model.MovieSession" />
```

Comment:
why did you have these lines of code in this PR? They should have already been merged